sqlite3.go

```diff
@@ -118,6 +119,7 @@ int compareTrampoline(void*, int, char*, int, char*);
 int commitHookTrampoline(void*);
 void rollbackHookTrampoline(void*);
 void updateHookTrampoline(void*, int, char*, char*, sqlite3_int64);
+void preUpdateHookTrampoline(void*, sqlite3 *, int, char *, char *, sqlite3_int64, sqlite3_int64);
```

Comment:
Please move this to a file named `sqlite3_opt_preupdate_hook.go` and set a build tag `sqlite_preupdate_hook`

Reply:
See `sqlite3_opt_userauth.go` as an example